### PR TITLE
fix: Improved performance when printing logs in Output Channel

### PR DESCRIPTION
**Pull request by Aerex**
# Description

When running a dev command through the VSCode command palette it takes longer than usual for the logs to appear in the Output panel. This PR solves this problem by upgrading the `vscode` npm package to 1.64 which fixes the buffering issues that occurs in the Output Channel.

# Callouts
- The `typescript` and `mocha` npm package were upgraded due to the upgrade to the `vscode` package
- There were some conditional logic changes to better ensure the Output Channel logs would appear (eg. `if(outputChannel !== undefined)` -> `if(outputChannel)`)
- As of now, colors are not suppored in the output channel so colors have been diabled in all platforms. See git issue [vscode#141556](https://github.com/microsoft/vscode/issues/141556) for more information.
- Updated travis build to run unit tests under docker 
- Used redhat ubi images to build base image for running vscode using xvfb (emulated X server)

# Steps to Reproduce
1. Download the latest version of the IBM Cloud CLI extension if not already
2. Open the Command Palette (Ctrl-Shift-P or F1) and enter the command ibmcloud dev build
3. Verify that it takes longer than a minute for the entire logs to appear in the Ouput Channel

# Solution
1. Prune any containers and images that was built in the **Steps to Reproduce** section
2. Install `vsce` npm package if not already: `npm install -g vsce`
3. Checkout PR branch
4. Run the command `vsce package`
5. Verify vsix package is built successfully
6. Open a starterkit project using VSCode
7. Go to **Extension -> Install From VSIX`** and select vsix file created in step 4

8. Reload VSCode when requested
9. Run `ibmcloud dev build` from Command Palette
10. Verify that performance for logs to appear have improved

### CODE REVIEW

**Comment by Aerex** [comment]
@steveclay @schmidtp0740  I was able to get the unit tests to run successfully in docker. FYI 

Ready for re-review

**Review by steveclay — APPROVED** [review]

**Comment by steveclay** [comment]
@schmidtp0740 please review the Travis / devops changes before we merge this. 👍

File: ci/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,83 @@
+FROM ubuntu:18.04
```
Comment (schmidtp0740) [inline ci/Dockerfile:1]: It would be best to use the redhat UBI images

Comment (schmidtp0740) [reply ci/Dockerfile:1]: Could start with https://catalog.redhat.com/software/containers/ubi8/ubi/5c359854d70cc534b3a3784e and then install python/node/etc. This image is always most up to date in the past when it comes to patching vulnerabilities

**Review by schmidtp0740 — CHANGES_REQUESTED** [review]

**Comment by Aerex** [comment]
@schmidtp0740 Updated docker build to create image using redhat UBI. Ready for re-review. FYI @steveclay

**Review by schmidtp0740 — APPROVED** [review]
Nice well done

**Pull request merged**